pyboy/utils.py

```diff
@@ -276,3 +277,48 @@ def __init__(
         self.mouse_scroll_x = mouse_scroll_x
         self.mouse_scroll_y = mouse_scroll_y
         self.mouse_button = mouse_button
+
+
+##############################################################
+# Memory Scanning
+#
+
+
+def dec_to_bcd(value, byte_width=1, byteorder="little"):
+    """
+    Converts a decimal value to Binary Coded Decimal (BCD).
+
+    :param value: Integer value to convert.
+    :param byte_width: The number of bytes to consider for each value.
+    :param byteorder: The endian type to use (see [int.from_bytes](https://docs.python.org/3/library/stdtypes.html#int.from_bytes)). Note, this is only used for 16-bit values and higher.
+    :return: BCD equivalent of the decimal value.
+    """
```

Comment:
The docs generator we're using doesn't pick up this format. You can see `pyboy.py` for examples on how to format it.